cmd/commands/flatten_rbd_pvc.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2024 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	storageProvisionerBetaAnnotation    = "volume.beta.kubernetes.io/storage-provisioner"
+	storageProvisionerAnnotation        = "volume.kubernetes.io/storage-provisioner"
+	cephCSIRBDStorageProviderAnnotation = "volume.beta.kubernetes.io/storage-provisioner"
```

Comment:
this looks to be same as `storageProvisionerBetaAnnotation` do we need both?

Reply:
Oops, sorry. I meant not "volume.beta.kubernetes.io/storage-provisioner" but "ceph-ssd.rbd.csi.ceph.com" for `cephCSIRBDStorageProviderAnnotation`.